instrumentation/github.com/aws/aws-sdk-go/service/go.mod

```diff
@@ -0,0 +1,16 @@
+module go.opentelemetry.io/contrib/instrumentation/github.com/aws/aws-sdk-go/service
```

Comment:
Will we ever need to instrument things from the `github.com/aws/aws-sdk-go` itself? If so, would it make sense to have this module match that structure by being one level up?